.github/workflows/stockfish_test.yml

```diff
@@ -115,7 +115,7 @@ jobs:
 
       - name: Extract the bench number from the commit history
         run: |
-          git log HEAD | grep "\b[Bb]ench[ :]\+[0-9]\{7\}" | head -n 1 | sed "s/[^0-9]*\([0-9]*\).*/\1/g" > git_sig
+          git log HEAD | grep -o "\b[Bb]ench[ :]\+[1-9][0-9]\{5,9\}\b" | head -n 1 | sed "s/[^0-9]//g" > git_sig
```

Comment:
Not directly related to your change but `git log HEAD` can be simplified to just `git log`. (git log man page: "When no revision-range is specified, it defaults to HEAD".)